### ISSUE TITLE
Make the check for correct config more intelligent (parse what it knows into the config screen)

### ISSUE THREAD

**Issue by shaleh** [mbos/Pass4Win#93]
I had to uninstall and reinstall which asked the initial questions again.

**Comment by mbos** [comment]
Yeah this about the running which you opened a seperate issue over. The config questions are triggered if it doesn't find the config or the config is corrupt. Corrupt has a broad meaning, it could mean an essential bit of information is not there. It could be fixed so it tries what it knows.
